frontend/src/assets/js/layout.js

```diff
@@ -0,0 +1,13 @@
+const wrapper = document.querySelector('#wrapper');
+const navBar = document.querySelector('#mainNav');
+const navButton = document.querySelector('#menuToggler')
+
+const navBarExpand = (e) => {
+    wrapper.classList.toggle('toggled');
+    navButton.checked ^= true;
+    console.log('yes');
+}
+
+navBar.addEventListener('click', (e) => {
+    navBarExpand();
```

Comment:
```suggestion
    navBarExpand(e);
```
Takes a parameter, may be able to simplify even further with
```js
navBar.addEventListener('click', navBarExpand);
```
You'd have to check on how the lambda interacts with it...